### PR TITLE
Design improvement in login form

### PR DESCRIPTION
**Pull request by PiotrKowalski**

### CODE REVIEW

File: open_event/static/admin/css/material-custom.css [hunk 1]
```diff
@@ -7840,3 +7840,6 @@ hr.on-light {
 .shadow-z-5 {
   box-shadow: 0 27px 24px 0 rgba(0, 0, 0, 0.2), 0 40px 77px 0 rgba(0, 0, 0, 0.22);
 }
+.auth_required {
+  margin-bottom: -15px;
+}
```
Comment (rafalkowalski) [inline open_event/static/admin/css/material-custom.css]: @PiotrKowalski Please don't modify libraries css, You can add changes to file open-event.css

Comment (PiotrKowalski) [inline open_event/static/admin/css/material-custom.css]: I didn't know. I can change then

**Comment by rafalkowalski** [comment]
Cool

**Comment by leto** [comment]
@rafalkowalski what are we doing with this?

**Comment by rafalkowalski** [comment]
@leto I think, Changes should be separated to another file

**Comment by leto** [comment]
Closing, as this doesn't seem likely to get merged

**Pull request closed without merging**